shared/app.js

```diff
@@ -91,6 +80,35 @@ module.exports = Backbone.Model.extend({
 
   /**
    * @shared
+   *
+   * Initialize the `templateAdapter`, allowing application developers to use whichever
+   * templating system they want.
+   *
+   * We can't use `this.get('templateAdapter')` here because `Backbone.Model`'s
+   * constructor has not yet been called.
+   */
+  initializeTemplateAdapter: function(entryPath, attributes) {
+    if (this.options.templateAdapterInstance) {
+      this.templateAdapter = this.options.templateAdapterInstance;
+    } else {
+      var templateAdapterModule = attributes.templateAdapter || this.defaults.templateAdapter,
+      templateAdapterOptions = {entryPath: entryPath};
+
+      templateAdapterOptions.templateFinder = this.getTemplateFinder();
```

Comment:
I think that this will override the default `templateAdapter.templateFinder` in `rendr-handlebars` to be a `_.noop` maybe this should be a conditional thing like...

```js
if (_.isFunction(this.getTemplateFinder) && this.getTemplateAdapter !== _.noop) {
  templateAdapterOptions.templateFinder = this.getTemplateFinder();
}
```

I like the idea of leaving it as a noop here - that way we can easily see the override point, however i don't think this will work as expected if we aren't overriding with a custom templateFinder with `rendr-handlebars`